### PR TITLE
Add test for emstrip

### PR DESCRIPTION
**Pull request by dschuff**
2nd part of the fix for #16884

### CODE REVIEW

File: tests/test_other.py [hunk 1]
```diff
@@ -609,6 +610,14 @@ def test_emsize(self):
       output = self.run_process(cmd, stdout=PIPE).stdout
       self.assertContained(expected, output)
 
+  def test_emstrip(self):
+    self.run_process([EMCC, test_file('hello_world.c'), '-g', '-o', 'hello.js'])
+    output = self.run_process([LLVM_OBJDUMP, '-h', 'hello.wasm'], stdout=PIPE).stdout
+    self.assertContained('.debug_info', output)
+    self.run_process([emstrip, 'hello.wasm'])
+    output = self.run_process([LLVM_OBJDUMP, '-h', 'hello.wasm'], stdout=PIPE).stdout
+    self.assertContained('.debug_info', output)
```
Comment (sbc100) [inline tests/test_other.py:619]: Should this be `assertNotContained`?

File: tools/shared.py [hunk 2]
```diff
@@ -690,6 +690,7 @@ def get_llvm_target():
 LLVM_COMPILER = os.path.expanduser(build_llvm_tool_path(exe_suffix('llc')))
 LLVM_DWARFDUMP = os.path.expanduser(build_llvm_tool_path(exe_suffix('llvm-dwarfdump')))
 LLVM_OBJCOPY = os.path.expanduser(build_llvm_tool_path(exe_suffix('llvm-objcopy')))
+LLVM_OBJDUMP = os.path.expanduser(build_llvm_tool_path(exe_suffix('llvm-objdump')))
```
Comment (sbc100) [inline tools/shared.py:693]: If the tool is only needed during testing can you put it in tests/common.py (see WASM_DIS for example)

**Review by sbc100 — COMMENTED** [review]
lgtm % comments

File: tests/test_other.py (hunk 1 above)
Comment (dschuff) [reply tests/test_other.py]: lol yes and I wondered why this test was failing :rofl:

File: tools/shared.py (hunk 2 above)
Comment (dschuff) [reply tools/shared.py]: Done.

**Review by sbc100 — APPROVED** [review]

**Pull request merged**